src/fides/api/api/v1/endpoints/messaging_endpoints.py

```diff
@@ -323,13 +323,13 @@ def get_messaging_status(
             service_type=messaging_config.service_type,  # type: ignore
             secrets=secrets,
         )
-    except (ValueError, KeyError) as e:
-        logger.error(f"Invalid secrets found on {messaging_config.service_type.value} messaging configuration: {Pii(str(e))}")  # type: ignore
+    except ValidationError:
```

Comment:
Can we fold this into the other errors and have one `except` block? Looks like the behavior is identical (except the logger) for `ValueError`, `KeyError` and `ValidationError`